inductiva/utils/format_utils.py

```diff
@@ -86,7 +86,7 @@ def get_tabular_str(tabular_data: Union[Mapping[str, Iterable[Any]],
         }
         #if we have no headers data will be empty.
         #So, we want our original tabular_data
-        if not headers:
+        if headers:
```

Comment:
Based on your bug comment, then we should check if there are headers before generating the data, no?

Reply:
We kinda don't need it because the  `for X in headers` will do nothing.

Reply:
Still... better to check first, because for example this bug would have been avoided :D

Can the data be a Mapping and not have headers?